resources/public/api/conf/3.0/schemas/retrieveHighIncomeChildBenefitChargeSubmission/retrieve_high_income_child_benefit_charge_submission_response_schema.json

```diff
@@ -23,7 +22,8 @@
       "type": "integer",
       "description": "Number of children you received child benefit for. The value must be an integer greater than or equal to 1.",
```

Comment:
Please update to the latest description "Number of children for which you received child benefit. The value must be an integer between 1 and 99."